src/statement.cc

```diff
@@ -238,16 +237,18 @@ template <class T> T* Statement::Bind(const Napi::CallbackInfo& info, int start,
     if (start < last) {
         if (info[start].IsArray()) {
             Napi::Array array = info[start].As<Napi::Array>();
-            int length = array.Length();
+            uint32_t length = array.Length();
             // Note: bind parameters start with 1.
-            for (int i = 0, pos = 1; i < length; i++, pos++) {
+            int pos = 1;
```

Comment:
`pos` is of type `int` to match [one of the Field constructors](https://github.com/microsoft/vscode-node-sqlite3/blob/6e3d91ce1703481cf2f74b60cf38ec3c45132946/src/statement.h#L23)

Reply:
why was it changed from `unsigned short` to `int` in this PR ?

Reply:
[sqlite3_bind_parameter_index](https://github.com/microsoft/vscode-node-sqlite3/blob/6e3d91ce1703481cf2f74b60cf38ec3c45132946/src/statement.cc#L301) actually returns `int`.

Reply:
I changed the PR so that `pos` is int. The sqlite3 API returns an int and then [asks for an int back later](https://www.sqlite.org/c3ref/bind_blob.html), rather than a short or unsigned int.